assembl/static2/js/app/components/common/medias.jsx

```diff
@@ -0,0 +1,20 @@
+// @flow
+import React, { Fragment } from 'react';
+
+import { videoUtilities } from '../../utils/videoUtilities';
+
+export type Props = {
+  path: string
+};
+
+export const Medias = ({ path }: Props) => (
+  <Fragment>
+    {videoUtilities.pathIsVideoFile(path) ? (
+      <video src={path} controls preload="none" width="100%" /> // eslint-disable-line
```

Comment:
Currently, there are 3 supported video formats for the <video> element: MP4, WebM, and Ogg...
https://www.w3schools.com/tags/tag_video.asp